### PR TITLE
PE-7171-byte-price

### PR DESCRIPTION
**Pull request by fedellen**
This PR adds getting the token price for given bytes in one method/command

image

### CODE REVIEW

File: tests/turbo.node.test.ts [hunk 1]
```diff
@@ -337,6 +342,23 @@ describe('Node environment', () => {
       expect(fees).to.have.length(1);
     });
 
+    describe.only('getTokenPriceForBytes()', async () => {
```
Comment (fedellen) [inline tests/turbo.node.test.ts:345]:
```suggestion
    describe('getTokenPriceForBytes()', async () => {
```

File: src/cli/cli.ts [hunk 2]
```diff
@@ -99,6 +100,15 @@ applyOptions(
   await runCommand(command, price);
 });
 
+applyOptions(
+  program
+    .command('token-price')
```
Comment (fedellen) [inline src/cli/cli.ts:105]: TODO: needs docs on command and sdk method

File: src/common/payment.ts [hunk 3]
```diff
@@ -287,6 +288,33 @@ export class TurboUnauthenticatedPaymentService
     }
     return response;
   }
+
+  public async getTokenPriceForBytes({
+    bytes,
+  }: {
+    bytes: number;
+  }): Promise<{ tokenPrice: string; bytes: number; token: TokenType }> {
```
Comment (fedellen) [inline src/common/payment.ts:296]: nit: can share `TurboTokenPriceResponse`

File: README.md [hunk 4]
```diff
@@ -905,6 +920,20 @@ turbo price --value 1024 --type bytes
 turbo price --value 1.1 --type arweave
 ```
 
+##### `token-price`
+
+Get the current price from the Turbo Payment Service, denominated in the specified token, for uploading a specified number of bytes to Turbo.
+
+Command Options:
+
+- `--value <value>` - Byte value to get the token price for
```
Comment (arielmelendez) [inline README.md:929]: Is there a reason this parameter needs to be named `value`? Why not `byteCount`?

File: README.md [hunk 5]
```diff
@@ -390,6 +392,19 @@ const { winc, actualTokenAmount, equivalentWincTokenAmount } =
...
 ```
 
+#### `getTokenPriceForBytes({ bytes })`
+
+Get the current price from the Turbo Payment Service, denominated in the specified token, for uploading a specified number of bytes to Turbo.
+
+```typescript
+const turbo = TurboFactory.unauthenticated({ token: 'solana' });
+const { tokenPrice } = await turbo.getTokenPriceForBytes({
+  bytes: 1024 * 1024 * 100,
```
Comment (arielmelendez) [inline README.md:402]: Could this be `byteCount` rather than `bytes`? `bytes` could be interpreted as actual raw bytes like a buffer or something.

Comment (fedellen) [reply README.md:402]: 1db8ece..171eadd

File: README.md (hunk 4 above)
Comment (fedellen) [reply README.md]: 171eadd

**Review by arielmelendez — APPROVED** [review]

**Pull request merged**